nuxeo-features/rest-api/nuxeo-rest-api-test/src/test/resources/test-operation-getdocumentparent-contrib.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<component name="org.nuxeo.rest.api.operation.test.contrib" version="1.0">
+
+  <extension target="org.nuxeo.ecm.core.operation.OperationServiceComponent" point="event-handlers">
+    <handler chainId="mytest_EmptyDoc_opChain">
```

Comment:
Here you could use `Document.GetParent` as the chain you declared just do a redirection to it.